src/coverlet.msbuild.tasks/MSBuildLogger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using ILogger = Coverlet.Core.ILogger;
+
+namespace Coverlet.MSbuild.Tasks
+{
+    class MSBuildLogger : ILogger
```

Comment:
nit: for this simple wrapper to msbuild logger you should use Expression-bodied members for better readability https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members